src/LogExpert.Core/Classes/Columnizer/SquareBracketColumnizer.cs

```diff
@@ -142,13 +143,13 @@ public string[] GetColumnNames ()
         var i = 1;
         while (columnNames.Count < GetColumnCount())
         {
-            columnNames.Insert(columnNames.Count - 1, "Source" + i++.ToString());
+            columnNames.Insert(columnNames.Count - 1, $"Source{i++.ToString(CultureInfo.InvariantCulture)}");
```

Comment:
in the string interpolation, ToString is not necessary just use {(i++)}

Reply:
Done